buildSrc/src/main/java/org/elasticsearch/gradle/ElasticsearchDistribution.java

```diff
@@ -182,22 +184,30 @@ public void setFlavor(Flavor flavor) {
         this.flavor.set(flavor);
     }
 
-    public boolean getBundledJdk() {
+    public boolean hasBundledJdk() {
         return bundledJdk.getOrElse(true);
     }
 
     public void setBundledJdk(Boolean bundledJdk) {
         this.bundledJdk.set(bundledJdk);
     }
 
-    public boolean getFailIfUnavailable() {
+    public boolean shouldFailIfUnavailable() {
```

Comment:
Again, this isn't only a description of the distribution, we configure this to _ask_ for a distribution. So in that context we are _telling_ the build to fail or not.

I think in this case we can just make this getter private. It's only used here locally and nothing else should be inspecting this properly. Whatever we do we should make the getter and setter the same name though, even if the getter is private.